dependencies.repos

```diff
@@ -5,7 +5,7 @@
 - git:
     local-name: tesseract
     uri: https://github.com/ros-industrial-consortium/tesseract.git
-    version: 0.16.1
+    version: 2a50d0f458f4f074ed465999521da51dc7159c30
```

Comment:
Lets use the master branch.

Reply:
I changed it to master. We need a new tag with the fixes in that repo at some point.